L3 Mathematical modelling of an electric vehicle /Calculating Energy Consumption {Software simulation in Scilab and Xcos} - Part 1.md

```diff
@@ -0,0 +1,200 @@
+- Class 2 – vehicles with 22 < PW~r~ <= 34
+- Class 3 – high-power vehicles with PW~r~ > 34
+
+Where *PW~r~ [kW/Tonne]* is the power-to-weight ratio, defined as the ratio between the rated engine power and kerb weight.
+
+
+![CodeCogsEqn.png](https://demo.pflms.com/markdown_attachments/1772/gzY8nxYPS7KIXjkQlVadbw)
+
+
+Our target is to convert the Honda Activa into a battery electric vehicle (BEV), and therefore we need to understand what is the current energy consumption of the vehicle. 
+
+From the previous task, we can extract the maximum power and kerb weight from the spec table and calculate the power-to-weight ratio:
+
+
+![power to weight ratio.png](https://www.pupilfirst.school/markdown_attachments/3232/g3RnVeYaWUFMMy4F1pj8-g)
+>Since the vehicle’s PW~r~ is bigger than 34, we are going to use the WLTC Class 3 driving cycle to calculate the energy consumption.
+
+
+![Speed-profile-for-WLTC-Class-3-driving-cycle.png](https://demo.pflms.com/markdown_attachments/1774/PJmsmJWVxpFEor6rbgbxIQ)*Image: Speed profile for WLTC Class 3 driving cycle*
+
+
+The parameters of the WLTC Class 3 cycle are summarized in the table below:
+
+|                                   | Low   | Medium | High  | Extra High | Total |
+|-----------------------------------|-------|--------|-------|------------|-------|
+| Duration, s                       | 589   | 433    | 455   | 323        | 1800  |
+| Stop duration, s                  | 150   | 49     | 31    | 8          | 235   |
+| Distance, m                       | 3095  | 4756   | 7162  | 8254       | 23266 |
+| % of stops                        | 26.5% | 11.1%  | 6.8%  | 2.2%       | 13.4% |
+| Maximum speed, km/h               | 56.5  | 76.6   | 97.4  | 131.3      |       |
+| Average speed without stops, km/h | 25.3  | 44.5   | 60.7  | 94.0       | 53.5  |
+| Average speed with stops, km/h    | 18.9  | 39.4   | 56.5  | 91.7       | 46.5  |
+| Minimum acceleration, m/s^2^        | -1.5  | -1.5   | -1.5  | -1.44      |       |
+| Maximum acceleration, m/s^2^        | 1.611 | 1.611  | 1.666 | 1.055      |       |
+
+The method to calculate the energy consumption is straightforward, and it makes use of the Scilab/Xcos simulation environment. 
+
+----
+
+>Everything explained in the video above is explained below as well, so that it is easier to follow and understand. if you are confused, please watch the video again. 
+
+## 1. Setting up the Simulation Environment
+
+We will now need to install and Setup Scilab, Follow the steps below to install and set it up.
+
+### Step 1. Download Scilab
+
+Click [here](https://www.scilab.org/download/6.1.0) to go to Scilab website, under the *Download* section and choose your installation package. For windows operating systems the are two versions: 32 and 64-bit. Download the one matching your operating system specification.
+
+When the download is complete you’ll have on your disk drive a *.exe file (e.g. Scilab-5.4.0.exe)
+
+### Step 2: Run *.exe file
```

Comment:
These need not be headings since many of these do not have any explanation under it.